TACC-Cloud/hazmapper
# Add drop-down menu and back button.

## Overview: ##
Add navigation to welcome page button and bring back the dropdown project selector in the control bar. 

## PR Status: ##


## Related Jira tickets: ##

* [DES-1953](https://jira.tacc.utexas.edu/browse/DES-1953)

## Summary of Changes: ##
Added the ui parts to the control-bar component.

## Testing Steps: ##
1. Confirm that routes work properly with the drop-down selector.
2. Confirm that the back button brings the user back to the welcome page.

## UI Photos:
![Screenshot_2021-05-04 HazMapper](https://user-images.githubusercontent.com/9425579/117072856-b5629080-acf6-11eb-8c70-64efae043d75.png)

## Notes: ##
